functions/source/ConfigParser/crhelper/resource_helper.pyi

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Callable, Optional, Union
+
+from crhelper import log_helper as log_helper
+
+logger: Any
+SUCCESS: str
+FAILED: str
+
+_DecoratorTypeDef = Callable[[Any, Any], Union[bool, str, None]]
+
+class CfnResource:
+    Status: str = ...
+    Reason: str = ...
+    PhysicalResourceId: str = ...
+    StackId: str = ...
+    RequestId: str = ...
+    LogicalResourceId: str = ...
+    Data: Any = ...
```

Comment:
is this needed?

Reply:
All of these `.pyi` files are needed as part of the library as they define the type interfaces used by various python tooling for static type-checking... I know I know, python strict typing is messy, but it's what we've got.

If you want to see all these `crhelper` libraries gone, we could include something in the build step to inject them later when the `functions/package/...` files are created. That or wrap them up as a lambda layer. But it's quite a bit simpler at the moment to keep them in as part of source code.